bash/README.md

```diff
@@ -248,28 +248,86 @@ barcode file:
+not found.
+
 ### Step 4b - Organize files
 
-In the out directory, make sure the files are organized by species. By
-default, files are sent to `~/scratch/demux`, but each file needs to be
-sent to `~/scratch/demux/<SPP>`. For example, the file
+In a new directory, make sure the files are organized by species. By
+default, files are sent to `~/scratch/demux/*sublibrary_name*`, but each
+file will need to be sent to `~/scratch/demux/*SPP*`. For example, the file
 “DS.MN.L01-DS.M.1.1.fq.gz” should be sent to the `~/scratch/demux/DS`
 directory.
 
 Note: this is not automated at this point but it would be nice to
 automate the file moving process so it’s not forgotten at this point.
 
+### Step 4c - Assess the raw, processed, and cleaned data
+
+In the script for [Step 4](#step-4---04-process_radtagssh), 
+we have specified that a new output folder will be created for each sublibrary.
+The output folder is where all sample files and the log file will be dumped for each
+sublibrary. It is important to specify a different output folder if you have multiple
+sublibraries because we will be assessing the output log for each sublibrary individually
+(and otherwise, the log is overwritten when the script loops to a new sublibrary).
+
+The utility `stacks-dist-extract` can be used to extract data from the log file.
+First, we examined the library-wide statistics to identify sublibraries
+where barcodes may have been misentered or where sequencing error may have occurred. 
+We used:
+
+     stacks-dist-extract process_radtags.log total_raw_read_counts
+     
+to pull out data on the total number of sequences, the number of low-quality reads, 
+whether barcodes were found or not, and the total number of retained reads per sublibary. 
+Look over these to make sure there are no outliers or sublibraries that need to be checked 
+and rerun.
+
+Next, we used:
+
+     stacks-dist-extract process_radtags.log per_barcode_raw_read_counts | head
+     
+to analyze how well each sample performed. There are three important statistics to
+consider for each sample. 
+1. *The proportion of reads per sample for each sublibrary* indicates the proportion 
+that each individual was processed and sequenced within the overall library. This is
+important to consider as cases where a single sample dominates the sublibary may
+indicate contamination.
+2. *The number of reads retained for each sample* can be an indicator of coverage.
+It is most likely a good idea to remove samples with a very low number of reads.
+Where you decide to place the cutoff for outliers is dependent on your dataset.
+For example, a threshold of 1 million reads is often used but this is not universal.
+3. *The proportion of reads retained for each sample* can also indicate low-quality
+samples and will give an idea of the variation in coverage across samples.
+
+The script `04-process_radtags_stats.R` was used to create plots for easily
```

Comment:
Where is this script? We should add it in :)

Reply:
Will add this in!